Sources/PackageDescription/PackageRequirement.swift

```diff
@@ -215,18 +230,29 @@ extension Range {
     /// version and going up to the next minor version.
     ///
     /// - Parameters:
-    ///     - version: The minimum version for the version range.
+    ///   - version: The minimum version for the version range.
     public static func upToNextMinor(from version: Version) -> Range<Bound> where Bound == Version {
         return version ..< Version(version.major, version.minor + 1, 0)
     }
 }
 
 @available(_PackageDescription, deprecated: 5.6)
 extension Package.Dependency.Requirement {
+    /// Returns a requirement for a version range, starting at the given minimum
+    /// version and going up to but not including the next major version. This is the recommended
+    /// version requirement.
+    ///
+    /// - Parameters:
+    ///   - version: The minimum version for the version range.
```

Comment:
It's [recommended](https://www.swift.org/documentation/api-design-guidelines/#write-doc-comment) to begin with a summary consisting only of a single sentence fragment. I would suggest something like this:

```
/// Returns a source control requirement bounded to the given version's major version number.
///
/// Additional descriptions and explanations here.
/// - Parameter version: The given version to bound the source control requirement to.
/// - Returns: A source control requirement bounded to `version`'s major version number.
```